matminer/featurizers/base.py

```diff
@@ -393,3 +392,65 @@ def citations(self):
 
     def implementors(self):
         return list(set(sum([f.implementors() for f in self.featurizers], [])))
+
+
+class StackedFeaturizer(BaseFeaturizer):
+    """Use the output of a machine learning model as features
+
+    For regression models, we use the single output class.
+
+    For classification models, we use the probability for the first N-1 classes where N is the
+    number of classes.
+    """
+
+    def __init__(self, featurizer=None, model=None, name=None, class_names=None):
+        """Initialize featurizer
+
+        Args:
+            featurizer (BaseFeaturizer): Featurizer used to generate inputs to the model
+            model (BaseEstimator): Fitted machine learning model to be evaluated
+            name (str): [Optional] name of model, used when creating feature names
+            class_names (str): Required for classification models, used when creating
```

Comment:
is class_names an array? doc lists as str

Also if an array, does it need to be in any special order?

Reply:
Oops. It is an array, I'll clarify that and the order.